cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -84,6 +84,55 @@ constexpr inline auto is_supported_construction_value_type()
 // Helper functions for `fixed_point` type
 namespace detail {
 
+/**
+ * @brief Helper struct containing common constants for setting and extracting
+ * the components of a floating point value.
+ */
+template <typename FloatingType,
+          typename cuda::std::enable_if_t<cuda::std::is_floating_point_v<FloatingType>>* = nullptr>
+struct FloatingComponentConstants {
```

Comment:
We use snake_case for types, but I'm not that familiar with this part of the code base

Reply:
template parameters use CamelCase but you are correct the struct name should be snake_case.